cmd/snap-update-ns/bootstrap.c

```diff
@@ -172,11 +281,19 @@ void bootstrap(void)
         return;
     }
 
-    // Look for known offenders in the snap name (.. and /) and do nothing if
-    // those are found. The golang code will validate snap name and print a
-    // proper error message but this just ensures we don't try to open / setns
-    // anything unusual.
-    if (partially_validate_snap_name(snap_name) < 0) {
+    // Ensure that the snap name is valid so that we don't blindly setns into
+    // something that is controlled by a potential attacker. Note that this no
+    // longer does partial validation as it did before.
```

Comment:
I think you can get rid of the 'Note that...'. The comment is sufficient in saying 'Ensure that the snap name is valid...'.

Reply:
Done